### ISSUE TITLE
如何设置loss func

### ISSUE THREAD

**Issue by yc930401** [liuqi6777/llm4ranking#3]
您好，非常好的开源项目，感谢感谢。

请问论文中提到的， For logits-based models，用llm4ranking/training/logits/train.py 这个训练，而且可以指定loss_func。在代码中，我没找到llm4ranking/training/logits/train.py 这个文件，在pointwise/train.py中我看到loss func好像是写死的cross entropy loss，我想指定rank net loss，请问需要怎么做呢？
